guide/content/deployment-steps.md

```diff
@@ -42,6 +44,8 @@ resources:
         parameter_value: 'false'  # Set to 'true' to enable CloudTrail integration
       - parameter_key: pEnableGuardDuty
         parameter_value: 'false'  # Set to 'true' to enable GuardDuty integration
+      - parameter_key: pDisableGuardDuty
```

Comment:
These seem inconsistent with similar values above that are `true/false` vs this as `yes/no`

Maybe it's because the submodules were written that way, but is there any way to make this `true/false` instead and translate that to `yes/no` later in the code? It just looks inconsistent.

But if it has to be this way please let me know.